Source/src/components/Component.h

```diff
@@ -22,6 +28,7 @@ namespace Hachiko
             DIRLIGHT,
             POINTLIGHT,
             SPOTLIGHT,
+            SCRIPT,
```

Comment:
Maybe this should be placed at the end of the enum to avoid problems while saving because it saves components type by this ID and it may break some scenes with UI. @Bernatmago correct me if I'm wrong

Reply:
Oh thanks, that must be lost in merging as I had to merge a lot of commits from `development` branch after I finished stuff. I'm gonna fix this.

Reply:
> Maybe this should be placed at the end of the enum to avoid problems while saving because it saves components type by this ID and it may break some scenes with UI. @Bernatmago correct me if I'm wrong

Ur totally right

Reply:
Made.